capture/parsers/udp.c

```diff
@@ -61,15 +61,15 @@ int udp_pre_process(ArkimeSession_t *session, ArkimePacket_t *const packet, int
     packet->direction = (dir &&
                          session->port1 == ntohs(udphdr->uh_sport) &&
                          session->port2 == ntohs(udphdr->uh_dport)) ? 0 : 1;
-    session->databytes[packet->direction] += (packet->pktlen - packet->payloadOffset - 8);
+    session->databytes[packet->direction] += ntohs(udphdr->uh_ulen) - 8;
 
     return 0;
 }
 /******************************************************************************/
 int udp_process(ArkimeSession_t *session, ArkimePacket_t *const packet)
 {
     const uint8_t *data = packet->pkt + packet->payloadOffset + 8;
-    int            len = packet->payloadLen - 8;
+    uint16_t        len = (packet->pkt[packet->payloadOffset + 4] << 8 | packet->pkt[packet->payloadOffset + 5]) - 8;
```

Comment:
So this will need some kind of MIN call otherwise could cause a buffer overflow read if len is greater then packet size.

Reply:
Yes, good catch, I'll add it